api.swagger.yaml

```diff
@@ -13083,6 +13262,217 @@ components:
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/user-trimmed-schema'
+            timestamp_created:
+              description: 'This is the timestamp (in seconds) of when the query was created.'
+              type: integer
+              example: 1510941596
+            timestamp_updated:
+              description: 'This is the timestamp (in seconds) of when the query was last updated.'
+              type: integer
+              example: 1510941696
+            comment_count:
+              description: 'Count of comments on the query.'
+              type: integer
+              example: 5
+            likes_count:
+              description: 'Count of likes on the query.'
+              type: integer
+              example: 5
+            published:
+              description: 'Published status for the query.'
+              type: boolean
+              example: true
+    query-schema-update:
+      allOf:
+        - $ref: '#/components/schemas/query-schema-common'
+        - type: object
+          properties:
+            attachment:
+              description: 'Query attachment.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the attachment.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+            query_assignee:
+              description: 'The assignee of this query.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the user.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+            query_status:
+              description: 'The query status.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the query status.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+            query_categories:
+              description: 'List of categories associated with the query.'
+              type: array
+              items:
+                type: object
+                properties:
+                  uuid:
+                    description: 'Invotra UUID of the query category.'
+                    type: string
+                    format: uuid
+                    example: '01234567-89ab-cdef-1234-56789abcdef0'
+            author:
+              description: 'The author of this query.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the user.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+    query-schema-create:
```

Comment:
These properties are not set during create by end users. End users fill out a webform which generates a query, administrators can then edit these fields.

We'll have to tie this in with the webform APIs